api_tests/preprints/views/test_preprint_detail.py

```diff
@@ -304,8 +304,18 @@ def test_update_preprints_with_no_type(self, app, user, preprint, url):
         res = app.patch_json_api(url, payload, auth=user.auth, expect_errors=True)
         assert res.status_code == 400
 
+    def test_update_preprints_with_no_type(self, app, user, preprint, url):
+        payload = {
+            'data': {
+                'id': preprint._id,
+            }
+        }
+
+        res = app.patch_json_api(url, payload, auth=user.auth, expect_errors=True)
+        assert res.status_code == 400
```

Comment:
Can you add an assertion that the error object is correct? It's possible for this to pass if an unrelated error is returned (e.g. missing attributes).